backend/src/graphql/model/User.ts

```diff
@@ -62,12 +69,28 @@ export class User {
   @Field(() => Int, { nullable: true })
   publisherId: number | null
 
-  @Field(() => Date, { nullable: true })
-  isAdmin: Date | null
-
   @Field(() => KlickTipp, { nullable: true })
   klickTipp: KlickTipp | null
 
   @Field(() => Boolean, { nullable: true })
   hasElopage: boolean | null
+
+  @Field(() => [String], { nullable: true })
+  roles: string[] | null
```

Comment:
yes, i understand your suggestion...
but the entity contains a nullable column userRoles and currently we only will have an associated userRole in case of admin or moderator. all other roles will not be stored in database nor they will be transferred in the graphql-model to the frontend.

So i would prefer to keep it for this role-stage like it is and as soon we want to support more detailed roles or perhaps serveral roles at one time, then we should overthink about it.